mongodb/intro/querying-arrays/querying-an-array-ii.md

```diff
@@ -0,0 +1,101 @@
+---
+author: Stefan-Stojanovic
+
+aspects:
+  - introduction
+
+type: normal
+
+category: must-know
+
+links:
+
+---
+# Querying An Array part 2
+---
+## Content
+
+In the previous insight, we showed you how to search for a specific element within an array regardless of its position in the array and the number of values in that array.
+
+```js
+db.pokedex.find( { spells: "Bite" } )
+```
+
+Now, let's say we wanted to find which document has an array with an exact number and order of elements.
+
+Using the same example from the previous insight, let's say we wanted to find which document contains only the "Bite" spell. 
+
+To do this, all we have to do is add `[]` around our `"Bite"` value, like so:
+```js
+// Search for an array containing the "Bite" value
+db.pokedex.find( { spells: "Bite" } )
+
+// Search for an array with only the specified value
```

Comment:
Match it with the comment above and use `"Bite"` instead of the generic "specified value":

```suggestion
// Search for an array containing only the "Bite" value
```